tests/protractor/chat/chat.js

```diff
@@ -42,7 +40,7 @@ app.controller('ChatCtrl', function Chat($scope, $firebase) {
       $scope.message = "";
 
       // Increment the messages count by 1
-      numMessagesSync.$transaction(function (currentCount) {
+      numMessagesFirebaseRef.transaction(function (currentCount) {
```

Comment:
Yeah, this should just be removed since it no longer tests AngularFir. No point making the tests more flaky by adding unnecessary transactions.